### DISCUSSION TITLE
Сессии ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#57] [Announcements]
# Сессии ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/sessions/

**Comment by Bahurtsev** [comment]
Первое это то что модуль 'redis' устарел и написанное выше не работает. Второе я не пойму, то есть чтобы в redis хранилище что-то записать, то надо установить это хранилище?  Потому что пишет ошибку The client is closed.

**Reply by Bahurtsev** [reply]
Ну да, действительно без базы данных Redis ничего не работает. А вы не слова не написали про то что её надо скачать и установить. В общем эта тема ввела меня в заблуждение на 3 дня. Я потратил 3 дня чтобы понять чтобы понять что это мне на надо. Какой смысл устанавливать сторонние базы данных если я уже пользуюсь MySql или MongoDB, поэтому я установил модуль 'connect-mongo' и с помощью него сессии автоматически летят в базу данных MongoDB.

**Reply by bndby** [reply]
Потому что редис гораздо эффективнее и быстрее для хранения сессий, чем монго или мускуль

**Reply by NikolaiBelski** [reply]
Вдвоем с парнем сайт штудируем))))
